.bumpversion.cfg

```diff
@@ -0,0 +1,7 @@
+[bumpversion]
```

Comment:
We can move this into `setup.cfg` if we'd like but the same issues with the tox.ini and pytest.ini files being rolled into there apply as well.